client/jetpack-connect/style.scss

```diff
@@ -915,8 +915,14 @@ $colophon-height: 50px; // wpcomColophon element at the bottom
 			margin-right: auto;
 			margin-left: auto;
 			padding-left: 0;
-			margin-top: 15px;
 			display: block;
+			height: 56px;
+			border-bottom: 1px solid #e1e2e2;
+			background: #fff;
```

Comment:
Please use CSS colour variables instead of hex codes.

See other files for examples or https://github.com/Automattic/wp-calypso/blob/3e69a83a497e8e2c0708d074311398cd98037df0/packages/calypso-color-schemes/src/shared/_color-schemes.scss for the full list.

E.g. this would be `var( --color-white )`

Reply:
Thanks for catching these two that I missed. Updated.